### ISSUE TITLE
Accessibility: Dropdowns and multiselects page issues

### ISSUE THREAD

**Issue by chosak** [cfpb/design-system#1109]
The [Dropdowns and multiselects](https://cfpb.github.io/design-system/components/dropdowns-and-multiselects) page has some accessibility issues that are [failing Lighthouse](https://github.com/cfpb/design-system/runs/1439607261?). See the Lighthouse report [here](https://storage.googleapis.com/lighthouse-infrastructure.appspot.com/reports/1606090464659-94313.report.html). The problems are:

![image](https://user-images.githubusercontent.com/654645/99975083-c4b8d780-2d6f-11eb-9a41-767d8d318646.png)

@anselmbradford I believe these changes may have been introduced in your PR #1096; would you mind taking a look to see how big of a fix this is?

**Comment by anselmbradford** [comment]
Yea sounds right. We wanted to bring the multiselect into the DS wholesale, improve it, and then deliver it back to cf.gov. So these should be issues that would be expected to fail on the blog filter too.

**Comment by stephanieosan** [comment]
ANDY FIXED THIS! :boom:

**Comment by chosak** [comment]
For reference that was this PR: https://github.com/cfpb/design-system/pull/1121 that @anselmbradford and I collaborated on.
